index.js

```diff
@@ -159,16 +151,3 @@ Level.destroy = function (db, callback) {
     callback(err)
   }
 }
-
-var checkKeyValue = Level.prototype._checkKeyValue = function (obj, type) {
```

Comment:
This method was dead (it's now called `_checkKey` in `abstract-leveldown`), but without the need for Array Buffers, the default works fine. Except for https://github.com/Level/abstract-leveldown/issues/230, but that's hardly a priority.